Doc/library/string.rst

```diff
@@ -59,11 +59,19 @@ The constants defined in this module are:
    String of ASCII characters which are considered punctuation characters
    in the ``C`` locale: ``!"#$%&'()*+,-./:;<=>?@[\]^_`{|}~``.
 
+
 .. data:: printable
 
-   String of ASCII characters which are considered printable.  This is a
-   combination of :const:`digits`, :const:`ascii_letters`, :const:`punctuation`,
-   and :const:`whitespace`.
+   String of ASCII characters which are considered printable by Python.
+   This is a combination of :const:`digits`, :const:`ascii_letters`,
+   :const:`punctuation`, and :const:`whitespace`.
+
+   .. note::
+
+      By design, :meth:`string.printable.isprintable() <str.isprintable>`
+      returns :const:`False`. In particular, ``string.printable`` is
+      neither POSIX printable (see :manpage:`LC_CTYPE <locale(5)>`)
+      nor printable in the sense of :rfc:`5822`.
```

Comment:
Are you sure about the RFC number? https://www.rfc-editor.org/info/rfc5822 says "Not Issued".

Reply:
I should have checked beforehand (I picked it up from the issue). The correct RFC is https://datatracker.ietf.org/doc/html/rfc1522 I think.

Reply:
Nevermind, it's https://datatracker.ietf.org/doc/html/rfc2047 which obsoletes 1522...

Reply:
... I hate those RFCs which refer to each others. Just wait a bit until I find the correct one which has the definition.

Reply:
I decided to leave out the mention to the RFC. The RFC is not per se a definition of what printable is but rather what printable means for the RFC itself and for headers in general.